packages/garbo/src/tasks/dailyItems.ts

```diff
@@ -408,6 +409,35 @@ const DailyItemTasks: GarboTask[] = [
     },
     spendsTurn: false,
   },
+  {
+    name: "Burning Leaves lit leaf lasso",
+    ready: () =>
+      BurningLeaves.have() &&
+      itemAmount($item`inflammable leaf`) >=
+        BurningLeaves.burnFor.get($item`lit leaf lasso`)!,
+    completed: () => get("_leafLassosCrafted") >= 3,
+    do: () => {
+      BurningLeaves.burnLeaves(
+        BurningLeaves.burnFor.get($item`lit leaf lasso`)!,
+      );
+    },
+    limit: { skip: 3 },
+    spendsTurn: false,
+  },
+  {
+    name: "Burning Leaves day shortener",
+    ready: () =>
+      BurningLeaves.have() &&
+      itemAmount($item`inflammable leaf`) >=
+        BurningLeaves.burnFor.get($item`day shortener`)!,
+    completed: () => get("_leafDayShortenerCrafted"),
+    do: () => {
+      BurningLeaves.burnLeaves(
+        BurningLeaves.burnFor.get($item`day shortener`)!,
+      );
```

Comment:
I think `BurningLeaves` has a function where you can directly pass the item in, and avoid non-null assertions